dnsext-dnssec/DNS/SEC/PubKey.hs

```diff
@@ -0,0 +1,70 @@
+module DNS.SEC.PubKey where
+
+import DNS.Types
+import DNS.Types.Internal
+import qualified DNS.Types.Opaque as Opaque
+
+import DNS.SEC.Imports
+import DNS.SEC.PubAlg
+
+data PubKey = PubKey_RSA { rsa_size :: Int, rsa_e :: Opaque, rsa_n :: Opaque }
+            | PubKey_ECDSA { dsa_x :: Opaque, dsa_y :: Opaque }
```

Comment:
I think `ec_x` and `ec_y` are clearer names. what do you think?

Reply:
How about `ecdsa_x` and `ecdsa_y`?

Reply:
I think that's good too